apps/website/build-scripts/video-loader.cjs

```diff
@@ -262,9 +274,13 @@ const videoLoader = async (context, content) => {
 };
 
 module.exports = function (content) {
+  console.log(`Processing video ${this.resourcePath}...`);
   const callback = this.async();
   videoLoader(this, content)
-    .then((res) => callback(null, res))
+    .then((res) => {
+      console.log(`Processed video ${this.resourcePath}: ${res}`);
```

Comment:
@pjeweb how do you feel about this logging? Is logging the full output too noisy? Is any level of logging too noisy?

Reply:
I dont mind it. We can remove it later if it bothers us. (or maybe add a environment variable to allow building w or w/o verbose build logs?)